ui/app/templates/vault/cluster/access/namespaces/index.hbs

```diff
@@ -35,7 +35,7 @@
           {{#let (concat this.currentNamespace (if this.currentNamespace "/") list.item.id) as |targetNamespace|}}
             {{#if (includes targetNamespace this.accessibleNamespaces)}}
               <li class="action">
-                <NamespaceLink @targetNamespace={{targetNamespace}} @class="is-block">
+                <NamespaceLink @targetNamespace={{targetNamespace}} @unparsed={{true}} @class="is-block">
```

Comment:
would you mind adding documentation to the NamespaceLink component? Add `unparsed` parma, etc.

Reply:
I'll add a comment about unparsed, but there's a lot in there that I have no idea what's going on 😬